.github/workflows/tag-release-quay.yml

```diff
@@ -14,6 +14,7 @@ env:
 jobs:
   copy-tag-to-quay:
     runs-on: ubuntu-latest
+    if: 'startsWith(${{ github.event.repository.name }}, "opendatahub-io")'
```

Comment:
```suggestion
    if: github.repository == 'opendatahub-io/data-science-pipelines-operator'
```

Reply:
sure, if this makes more clear with intention !